pollux/lib/sql-doobie/src/main/scala/io/iohk/atala/pollux/sql/repository/JdbcCredentialStatusListRepository.scala

```diff
@@ -172,6 +172,29 @@ class JdbcCredentialStatusListRepository(xa: Transactor[ContextAwareTask], xb: T
 
   }
 
+  def revokeByIssueCredentialRecordId(
+      issueCredentialRecordId: DidCommID
+  ): RIO[WalletAccessContext, Boolean] = {
+
+    for {
+      walletId <- ZIO.service[WalletAccessContext].map(_.walletId)
+      updateQuery =
+        sql"""
+             | UPDATE public.credentials_in_status_list AS cisl
+             | SET is_canceled = true
```

Comment:
If we only support revocation, then I propose to rename this column to `is_revoked` for better clarity.

A related question is, do we want to support suspension in the future? If yes (and I guess we will), then why not name it `status` and use an enum with possible values: `VALID`, `REVOKED`, and `SUSPENDED`? Fixing this once it has been deployed will be more complex and imply migration scripts...

Reply:
Ok, I see. I thought the actual status (revoked or suspended) was stored in a specific DB column while it is actually stored in the encoded VC status list. Using a DB query, one can only know if a VC is `valid` or has been `cancelled`, but not the reason (suspension or revocation).